app/src/main/java/com/capstone/weatherapp/CityListViewModel.kt

```diff
@@ -3,7 +3,7 @@ package com.capstone.weatherapp
 import androidx.lifecycle.ViewModel
 
 class CityListViewModel : ViewModel() {
-    private var _list = ArrayList<Int>(List(100) { it + 1 })
-    val list: ArrayList<Int>
+    private var _list = ArrayList<City>(List(100) { City("New York", 72) })
```

Comment:
Instead of an ArrayList that holds type City, let's make this a regular list that holds Ints (since we don't need the City.kt class anymore)